tests/unit/test_locationcodes.py

```diff
@@ -0,0 +1,77 @@
+"""Tests for strtools.py"""
+
+import time
+import timeit
+import importlib
+
+import pytest
+from scipy import stats
+
+import zolltools.nemsis.locationcodes as locationcodes
+
+@pytest.mark.slow
+def test_get_mapping_performance():
+    """
+    measuring performance of get_mapping
+    
+    Takes a sample of the execution time for the first and successive calls to
+    `get_mapping` and determines that successive calls are at least
+    `min_exp_speedup` times faster than the first calls within a certain
+    confidence interval (see `alpha` and comparison to p-value in `assert`
+    statement).
+    """
+
+    alpha = 0.05             # confidence threshold
+    min_exp_speedup = 40     # successive reads should be min_exp_speedup times faster
```

Comment:
Add explanation for this number. Why is it 40?

Reply:
This number was determined with preliminary testing, and I will explaining the reasoning here. The docstring will reference this PR so that this explanation can be viewed when necessary. I do not think this explanation should go in the docstring.

When the measurement for successive reads is taken as an average of 1,000,000 runs (using `timeit`) the speedup is about 100x. When the number of repetitions is reduced to ~1,000 and tested in this format, an expected speedup of 50x will sometimes fail. Therefore, I set the lower bound for the speedup to 40x, which is still enough to detect a significant change in the speed of successive reads. This test, with an expected speedup of 40x, is enough to detect the issue we are testing for, that is, that the mapping was being repeatedly read from memory.